www/docs/en/dev/guide/platforms/ios/index.md

```diff
@@ -69,10 +69,20 @@ $ xcode-select --install
 The [ios-deploy](https://www.npmjs.org/package/ios-deploy) tools allow you
 to launch iOS apps on an iOS Device from the command-line.
 
-To install it, run the following from command-line terminal:
+Install ios-deploy via [Homebrew](https://brew.sh/) by running:
 
 ```bash
-$ npm install -g ios-deploy
+$ brew install ios-deploy
```

Comment:
What's wrong with installing `ios-deploy` from npm?

Reply:
On macOS there is an error if you try to install it from npm.

According to the main page of ios-deploy, please refer to new installation guide:
https://github.com/ios-control/ios-deploy#Installation

Reply:
And also you can refer to installation guide at npm main page for the ios-deolpy:
https://www.npmjs.com/package/ios-deploy#installation

Reply:
```
$ brew info ios-deploy
ios-deploy: stable 1.10.0 (bottled), HEAD
```
`brew` has version `1.10.0` released.

```
$ npm info ios-deploy
ios-deploy@1.11.0 | GPLv3 | deps: none | versions: 41
```
`npm` has version `1.11.0` released.

Just for information, `npm` is newer.